index.html

```diff
@@ -7,9 +7,27 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+ <header class="book-header"> 
+ <h1 class="book-title">Título del libro</h1>
+ <p class="book-genre">(género)</p>
+ </header>
+ <div class = "book-content">
```

Comment:
Try not to add spaces between attribute name and its value
`class="book-content"` looks better